pre_commit/checkers/syntax_ruby_checker.rb

```diff
@@ -33,8 +34,9 @@ def examine_code
 
   private
 
-  def disabled_via_preference?
-    PreCommitHelper.disabled_via_preference?(HOOK_KEY, @force_pref_on)
+  def self.use_for_project?
```

Comment:
`private` (on line 35) does not make singleton methods private. Use `private_class_method` or `private` inside a `class << self` block instead.